spot-client/src/spot-remote/ui/components/dial-pad/dial-pad.js

```diff
@@ -244,6 +262,28 @@ export class DialPad extends React.Component {
     }
 }
 
+/**
+ * Creates actions which can update Redux state.
+ *
+ * @param {Function} dispatch - The Redux dispatch function to update state.
+ * @private
+ * @returns {Object}
+ */
+function mapDispatchToProps(dispatch) {
+    return {
+        /**
+         * Display an app notification.
+         *
+         * @param {string} type - The type of the notification to display.
+         * @param {string} message - The text to display in the notification.
+         * @returns {void}
+         */
+        onAddNotification(type, message) {
```

Comment:
I think conceptually this prop should be called something like onPhoneValidationError, and not mention something about a notification because that's an implementation detail that doesn't need to be stated in the prop name.

Reply:
ok